src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -454,6 +454,10 @@ def notifySlackBuildFailure(exception, channel) {
 }
 
 def versionHasIncremented(currVers, newVers) {
+  // first version is valid, i.e. currVers will be empty
+  if (currVers == '') {
```

Comment:
I think this check would pass in the scenario of the version having been removed - is that the case?

Reply:
This function is only used internally in the library, so the first parameter will be the current version in the master branch of the repository. In theory this is only possible on initial repo creation (i.e. there is no package.json, so no version at all), as this code is intended for invalid versions getting into the master branch.

Reply:
Ah yeah, current version. Once the file is in master it will be ok. My bad - this check is ok.